src/selectors.js

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
"selectors" seems an odd name choice, esp as it's overloaded per CSS selectors.  Maybe these could just be in CognitoUtils?

Reply:
Selectors is a term of art in Redux: https://medium.com/@matthew.holman/what-is-a-redux-selector-a517acee1fe8